File: app/src/main/java/com/mayulive/myapplication/MainActivity.java
package com.mayulive.myapplication;


import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;


public class MainActivity extends AppCompatActivity
{
	private static final String TAG = "MA";

	@Override
	protected void onCreate(Bundle savedInstanceState)
	{
		super.onCreate(savedInstanceState);

		setContentView(R.layout.activity_main);

		getSupportFragmentManager().beginTransaction()
				.add(R.id.fragment_container, new MyFragment() )
				.commitNow();

	}
}

File: app/src/main/java/com/mayulive/myapplication/MainActivity.kt
package com.mayulive.myapplication

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.GridLayoutManager

import kotlinx.android.synthetic.main.activity_main.*



@Suppress("MemberVisibilityCanBePrivate")
class MainActivity : AppCompatActivity() {

   // fun clear(@Suppress("UNUSED_PARAMETER") view: View) = console?.clear()

    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)

        val adapter = RecyclerAdapter();
        val manager = GridLayoutManager(this, 2);   // 2 wide, height is controlled in adapter

        this.recycler.adapter = adapter;
        this.recycler.layoutManager = manager;
    }
}